Packs/DomainTools_Iris/Scripts/DomainToolsCheckPivotableDomains/check_pivotable_domains.yml

```diff
@@ -0,0 +1,99 @@
+args:
+- name: domaintools_data
+  required: true
+- name: max_registrant_contact_name_count
+  defaultValue: "200"
+- name: max_registrant_org_count
+  defaultValue: "200"
+- name: max_registrar_count
+  defaultValue: "200"
+- name: max_ssl_info_organization_count
+  defaultValue: "200"
+- name: max_ssl_info_hash_count
+  defaultValue: "350"
+- name: max_ssl_email_count
+  defaultValue: "350"
+- name: max_ssl_subject_count
+  defaultValue: "350"
+- name: max_name_server_host_count
+  defaultValue: "250"
+- name: max_name_server_ip_count
+  defaultValue: "250"
+- name: max_name_server_domain_count
+  defaultValue: "250"
+- name: max_soa_email_count
+  defaultValue: "200"
+- name: max_ip_address_count
+  defaultValue: "200"
+- name: max_mx_ip_count
+  defaultValue: "200"
+- name: max_mx_host_count
+  defaultValue: "200"
+- name: max_mx_domain_count
+  defaultValue: "200"
+- name: max_google_adsense_count
+  defaultValue: "200"
+- name: max_google_analytics_count
+  defaultValue: "200"
```

Comment:
Please add descriptions to these arguments.

Reply:
updated!